### PR TITLE
fixed install path; added torchvision;

### PR DESCRIPTION
**Pull request by loretoparisi**
Just fixed the installation path and added `torchvision` to requirements!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Clone the Synchronized-BatchNorm-PyTorch repository for
 cd Face_Enhancement/models/networks/
 git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
 cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
-cd ../../../
+cd ../../
```
Comment (westscz) [inline README.md:31]: I'm pretty sure that we doesn't need this change. Take a look on l28.

File: requirements.txt [hunk 2]
```diff
@@ -6,4 +6,5 @@ dominate>=2.3.1
 dill
 tensorboardX
 scipy
-opencv-python
\ No newline at end of file
+opencv-python
+torchvision
```
Comment (westscz) [inline requirements.txt:10]: For what purpose we need this package here?

**Review by westscz — CHANGES_REQUESTED** [review]
I think that this PR is not needed and can be closed.

File: requirements.txt (hunk 2 above)
Comment (loretoparisi) [reply requirements.txt:10]: Because without it did not work, at least when I have tried to install locally.

File: README.md (hunk 1 above)
Comment (loretoparisi) [reply README.md:31]: After following the README.md on fresh instdall from the repo, that path was wrong.

**Pull request closed without merging**